### PR TITLE
implement core webhooks

### PR DESCRIPTION
**Pull request by capaj**

### CODE REVIEW

**Comment by PatrickHeneise** [comment]
Missing the x-hub-signature check ... @bpedro is working on the same thing and has it implemented already.

**Comment by iancrowther** [comment]
closing Bruno & Jiri need to contact each other

**Comment by bpedro** [comment]
Interesting approach (EventEmitter) but not performing the following checks:

* valid headers (x-hub-signature, x-github-event and x-github-delivery)
* signature (taken from x-hub-signature) is actually valid

File: integrations/gitup-webhook.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+var eventBus = new EventEmitter();
+
+/**
+ * use this for subscribing to events from github, supply express.js app, get a
+ * var webhook = require('./integrations/gitup-webhook.js')(app);
+ * webhook.on('issues', function(eventData){})
+ */
+module.exports = function(app) {
+	app.post('/github/delivery', function (req, res) {
+		var header = req.header('X-Github-Event');
```
Comment (bpedro) [inline integrations/gitup-webhook.js]: What happens when x-github-event header is not present?

Comment (capaj) [inline integrations/gitup-webhook.js]: yeah I haven't thought about a lot of stuff. We'll use your gitup-webhook

**Comment by iancrowther** [comment]
gents - is this still a live conversation?

**Comment by PatrickHeneise** [comment]
latest code is here: https://github.com/bpedro/gitup-webhook

**Comment by capaj** [comment]
@iancrowther no, dead

**Pull request closed without merging**